tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -510,6 +511,7 @@ TEST_F(ZoneIdealLoadsTest, IdealLoads_ExhaustNodeTest)
     state->dataHeatBal->space(1).HTSurfaceFirst = 1;
     state->dataHeatBal->space(1).HTSurfaceLast = 1;
     state->dataScheduleMgr->Schedule.allocate(1);
+    state->dataEnvrn->StdRhoAir = 1.0; // Prevent divide by zero
```

Comment:
Yep, we should initialize this to a meaningful value in the fixture, and let individual unit tests overwrite only if necessary.